### PR TITLE
[15.0][MIG] edi_backend_partner_oca

### PR DESCRIPTION
**Pull request by JaumeBforgeFlow**
Standard migration

### CODE REVIEW

File: edi_backend_partner_oca/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Edi Backend Partner",
```
Comment (simahawk) [inline edi_backend_partner_oca/__manifest__.py]:
```suggestion
    "name": "EDI Backend Partner",
```

File: edi_backend_partner_oca/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "summary": "add the a partner field in EDI backend",
```
Comment (simahawk) [inline edi_backend_partner_oca/__manifest__.py]:
```suggestion
    "summary": "Add a partner field to EDI backend",
```

File: edi_backend_partner_oca/__manifest__.py (hunk 1 above)
Comment (JaumeBforgeFlow) [reply edi_backend_partner_oca/__manifest__.py]: Thanks, done

File: edi_backend_partner_oca/__manifest__.py (hunk 2 above)
Comment (JaumeBforgeFlow) [reply edi_backend_partner_oca/__manifest__.py]: Done

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM! :+1: functional and code review

**Review by simahawk — APPROVED** [review]

**Pull request merged**